docs/advanced.rst

```diff
@@ -1562,3 +1562,27 @@ work, it is important that all lines are indented consistently, i.e.:
 
 .. [#f4] http://www.sphinx-doc.org
 .. [#f5] http://github.com/pybind/python_example
+
+Calling Python from C++
+=======================
+
+Pybind11 also allows to call python code from C++.
```

Comment:
More complete documentation would be desirable, see e.g. http://www.boost.org/doc/libs/1_38_0/libs/python/doc/v2/exec.html